### DISCUSSION TITLE
Output index.html in location dist/index.html

### DISCUSSION THREAD

**Discussion by AsuraKev** [web-infra-dev/modern.js#4658] [Q&A] [answered]
Hi,

As part of deployment to Azure static web app, the index.html needs to be on the root level folder. how do you output index.html directly under **dist** folder and not wrapped in a main folder? 

I have tried below but the html is still wrapped in a main folder. thanks :)

 distPath:{
      html: '.'
    }

**Comment by chenjiahan** [comment, marked as answer]
Try this:

```js
export default {
  source: {
    // The default is 'main'
    mainEntryName: 'index',
  },
  output: {
    distPath: {
      // The default is 'html'
      html: '',
    },
  },
  html: {
    disableHtmlFolder: true,
  },
};
```

**Reply by AsuraKev** [reply]
You save my day! thanks :)

**Comment by xmsz-stu** [comment]
有点麻烦
理论上，我应该只需要设置output.distPath.html = '', 但是居然没有生效...
还需要设置html.disableHtmlFolder = true, 为什么会互相影响？
设置以后，还是达不到效果，因为产物是 main.html，但是正常来说应该是index.html
这时候又要设置source.mainEntryName = 'index'
非常麻烦，和不符合直觉，不来issue找不到答案
